.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: python
 
-sudo: false
```

Comment:
Why did you remove sudo: false? Please revert.

Reply:
> Why did you remove sudo: false? Please revert.

In [November 2018 Traivs announced they would be retiring this API](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration). In the post they advise that

> If you currently specify `sudo: false` in your `.travis.yml`, we recommend removing that configuration soon.

Reply:
Thank you for noting this, but could you still please revert this, as it is not related to change you want to make? There should be only one feature change per PR, and these are two unrelated feature changes.

Reply:
I will remove the `sudo: false` later.

Reply:
@HDembinski Sure. I'll do so and rebase this. I totally appreciate the idea of wanting PRs to be as atomic as possible &mdash; the only reason I didn't do that here is it felt like an incredibly minor change given that the API is being dropped and not modified and so will have downstream effects.